dump_frames.py

```diff
@@ -53,10 +53,11 @@ def frames_already_dumped(video_path,
         # moviepy, so we have to explicitly check). We can assume they start
         # with index 1, and continue.
         offset_if_one_indexed = 1
+
     expected_frame_paths = [
         expected_name_format % (i + offset_if_one_indexed)
         for i in range(int(math.floor(expected_duration *
-                                      expected_frames_per_second)))
+                                      expected_frames_per_second) - 1))
```

Comment:
Hm, do you have an example video / invocation where the previous command fails? The current version seems to work for me for a couple test videos.

Reply:
Try this video with --fps=0, it will look for a 150th frame when FFmpeg only produces 149.

https://www.dropbox.com/s/ktehfyhu7ird168/jackson_hole.mp4?dl=0

Reply:
I see, that's annoying. Okay, let's go with this fix for now. Maybe later I can add a more robust way to compute the expected number of frames.